tests/versioned/apollo-server-express/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "apollo-server-tests",
   "version": "0.0.0",
   "private": true,
+  "engines": {
```

Comment:
Isn't this one superfluous? IIRC we only respect the `engines` stanza inside `tests`

Reply:
this is for eslint.  its default is 8 and i updated to use spread and catch without an error syntax